kyo-scheduler/jvm/src/main/scala/kyo/scheduler/InternalClock.scala

```diff
@@ -30,15 +29,9 @@ final private case class InternalClock(executor: Executor) {
 
     def currentMillis(): Long = millis
 
-    def stop(): Unit = {
-        gauge.close()
+    def stop(): Unit =
```

Comment:
The gauge will now be automatically collected when there are no strong references to it anymore.